gooddata-server-oauth2-autoconfigure/src/main/kotlin/OktaRegistrationProperties.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 GoodData Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.gooddata.oauth2.server
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+import org.springframework.boot.context.properties.bind.DefaultValue
+
+@ConstructorBinding
+@ConfigurationProperties(prefix = "spring.security.oauth2.client.registration.okta")
+class OktaRegistrationProperties(
```

Comment:
Why do you need it?

Reply:
Good catch, this was some left over from my initial experiments, removed!